mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala

```diff
@@ -232,6 +363,47 @@ class LogisticRegressionSuite
     }
   }
 
+  test("coefficients and intercept methods") {
+    val mlr = new LogisticRegression().setMaxIter(1)
+    val mlrModel = mlr.fit(smallMultinomialDataset)
+    val thrownCoef = intercept[SparkException] {
+      mlrModel.coefficients
+    }
+    val thrownIntercept = intercept[SparkException] {
+      mlrModel.intercept
+    }
+    assert(thrownCoef.getMessage().contains("use coefficientMatrix instead"))
+    assert(thrownIntercept.getMessage().contains("use interceptVector instead"))
+
+    val blr = new LogisticRegression().setMaxIter(1)
```

Comment:
In tests like these, perhaps we should explicitly set the family to be certain of the code path. We already have a test to check the family detection code. I'll add this in my next commit.